renderer/reducers/peripherals.ts

```diff
@@ -16,14 +23,14 @@ function getParams(peripheral) {
   return res;
 }
 
-const peripherals = (state = initialPeripheralState, action) => {
+const peripherals = (state: PeripheralState = initialPeripheralState, action) => {
   const nextState = Object.assign({}, state);
   const nextPeripherals = nextState.peripheralList;
   switch (action.type) {
-    case 'UPDATE_PERIPHERALS': {
+    case consts.PeripheralActionsTypes.UPDATE_PERIPHERALS: {
       const keys = [];
       action.peripherals.forEach((peripheral) => {
-        if (peripheral.device_type === PeripheralTypes.BatteryBuzzer) {
+        if (peripheral.device_type === consts.PeripheralTypes.BatteryBuzzer) {
           const batteryParams = getParams(peripheral);
           if (batteryParams.is_unsafe !== undefined) {
             nextState.batterySafety = batteryParams.is_unsafe;
```

Comment:
Somehow inside my VSC `batteryParams.is_unsafe` is redlining? I am not sure what is the problem? I didn't change the getParams function, and it seems that the function GetParams is interacting with sensor_data in fakeRuntime.

Reply:
Ah I see, this is because we have `const batteryParams = getParams(peripheral);` and `batteryParams` is an object but which we should probably make an interface for what values are in there. This is where types come in handy. But we'll probably have to dig around the code and see what actually belongs inside the interface.